app/assets/stylesheets/common/communicarts.scss

```diff
@@ -80,7 +80,19 @@ table td span.tag-values {
   font-weight: normal;
 }
 
+td.amount {
```

Comment:
rename this file while we're here? (there's a story somewhere re: moving away from communicart naming)